src/main/java/dev/esophose/playerparticles/particles/ParticleEffect.java

```diff
@@ -378,8 +378,8 @@ public static void display(ParticlePair particle, PParticle pparticle, boolean i
             effect.display(data, pparticle.getXOff(), pparticle.getYOff(), pparticle.getZOff(), 1, pparticle.getLocation(false), isLongRange, owner);
         } else if (effect.hasProperty(ParticleProperty.VIBRATION)) {
             Vibration data;
-            if (pparticle.getOverrideData() instanceof Vibration) {
-                data = (Vibration) pparticle.getOverrideData();
+            if (pparticle.getOverrideData() instanceof org.bukkit.Vibration) {
```

Comment:
Please double check that this all still runs in 1.16.5 and below. There were some really janky classloader issues around the bukkit vibration classes which is why that awful VibrationUtils class exists.